ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml

```diff
@@ -13,6 +13,10 @@ resources:
 tag_specification:
   name: "4.3"
   namespace: ocp
+test:
```

Comment:
There's no top-level `test` field, this belongs under the individual test item. (https://github.com/openshift/ci-tools/blob/master/CONFIGURATION.md#testsecrets)

Reply:
LOL, there's a mistake in that doc, I'll fix it.

Reply:
I have moved the secrets as you suggested, but the prowgen utility complains:
```
ERRO[0000] Failed to load CI Operator configuration      error="failed to load ci-operator config (error unmarshaling JSON: json: cannot unmarshal object into Go struct field TestStepConfiguration.tests.secrets of type []*api.Secret)" source-file=/home/lmilbaum/src/github.com/lioramilbaum/release/ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml
FATA[0000] Failed to generate jobs                       error="failed to load ci-operator config (error unmarshaling JSON: json: cannot unmarshal object into Go struct field TestStepConfiguration.tests.secrets of type []*api.Secret)" source=/home/lmilbaum/src/github.com/lioramilbaum/release/ci-operator/config subdir=openshift-cnv/cnv-ci target=/home/lmilbaum/src/github.com/lioramilbaum/release/ci-operator/jobs
```

Reply:
ignore. figured it out.

Reply:
That's likely because `secrets` is a YAML list, not an individual item.